journal/original/18-chapter-15.adoc

```diff
@@ -2679,6 +2714,7 @@ some under miserable huts and hovels, many without a rag or necessary utensils,
 bed or board; who, from delicateness, riches,
 or easy accommodations in stately and well-furnished houses,
 were now reduced to extremest misery and poverty,`" etc., vol. i. p. 372-374.]
+//Should the footnote end here ^^?
```

Comment:
Yes, sorry, this was before I fixed the footnote bug. Footnote ends there, and there are no more embedded docs below.